### ISSUE TITLE
Guidance/Comments sections not accessible

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#1444]
The guidance and comments section of the edit plan page is not accessible via keyboard. The user cannot reach these sections by tabbing through the page elements. The underlying accordions are also not accessible.

Make sure that the tabs and accordion/collapse elements have the proper roles or 'tablist' and 'tabs'

This feedback was provided from the UCOP electronic accessibility review. It was defined as having 'High' priority.

**Comment by johnpinto1** [comment]
@magdalenadrafiova @briri @xsrust
I don't think is an issue any more, correct if I am wrong. I am able to navigate to **guidance and comments section of the edit plan page**. Please see my screen shots:

![Selection_057](https://user-images.githubusercontent.com/8876215/61706851-0f23e180-ad41-11e9-9d20-a2dddc9298ee.png)
![Selection_058](https://user-images.githubusercontent.com/8876215/61706857-134fff00-ad41-11e9-9982-2ba4f3d25c34.png)

**Comment by sjDCC** [comment]
Hi @johnpinto1 I can also tab to comments and guidance but don't pick up secondary tabs (i.e. it doesn't open comments when a piece of guidance is displayed and I can't access second tab guidance)
![tab](https://user-images.githubusercontent.com/6169940/65811988-c52fff80-e1b8-11e9-8371-930d2d4243f3.PNG)


Is this user error or a tech issue to fix?

**Comment by johnpinto1** [comment]
@sjDCC An easy fix for this has been done. Awaiting PR to be acvcepted and merged.
![Selection_053](https://user-images.githubusercontent.com/8876215/69253785-181ca680-0bad-11ea-87df-712d8c98ab94.png)

**Comment by magdalenadrafiova** [comment]
@sjDCC may I ask for help with this one please?

**Comment by sjDCC** [comment]
If you "expand all" to open up each section then the tab works through all guidance / comments tabs and links
